### ISSUE TITLE
Missing ESLint rule disable in @typescript-eslint rule converter(s)

### ISSUE THREAD

**Issue by Res42** [typescript-eslint/tslint-to-eslint-config#1091]
### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 2.4.2
-   ESLint version: -
-   Node version: -

#### Actual Behavior

This issue was created according to this comment: <https://github.com/typescript-eslint/tslint-to-eslint-config/pull/1085#issuecomment-846487277>.

There are [@typescript-eslint/...](https://github.com/typescript-eslint/typescript-eslint) rules that require to turn off their `ESLint` counterpart like this:

```jsonc
{
  // note you must disable the base rule as it can report incorrect errors
  "no-use-before-define": "off",
  "@typescript-eslint/no-use-before-define": ["error"]
}
```
Source: <https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-use-before-define.md>

In the rule [convertNoUseBeforeDeclare](https://github.com/typescript-eslint/tslint-to-eslint-config/blob/main/src/converters/lintConfigs/rules/ruleConverters/no-use-before-declare.ts) this is not disabled.

Also I think it would be good to check all `@typescript-eslint` rule converters for this type of bug.

#### Expected Behavior

`@typescript-eslint` rule converters that require to turn off their `ESLint` rule counterpart actually turn them off.

**Comment by Res42** [comment]
| @typescript-eslint rule | ESLint rule |
| --- | --- |
| | |
| | |
| | |
| | |

**Comment by Res42** [comment]
A more authorative list can be found here <https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/README.md#extension-rules>.

**Comment by KingDarBoja** [comment]
Didn't know there was a list for these rules, glad you shared it 🚀
